test/stripe/subscriptions/subscription_item_test.exs

```diff
@@ -30,9 +30,8 @@ defmodule Stripe.SubscriptionItemTest do
   describe "delete/2" do
     test "deletes a subscription" do
       {:ok, subscription_item} = Stripe.SubscriptionItem.retrieve("sub_123")
-      assert {:ok, %{deleted: deleted, id: _id}} = Stripe.SubscriptionItem.delete("sub_123")
+      assert {:ok, %Stripe.SubscriptionItem{}} = Stripe.SubscriptionItem.delete("sub_123")
```

Comment:
I should clarify that this changed because stripe-mock now returns the "object" in the response (not sure of exact version), thus we convert to a struct with `id` populated and rest of keys nil, whereas before, we couldn't determine the struct, so we just processed the bare map.

Reply:
Meaning stripe-mock is returning `%{deleted: deleted, id: _id, object: object}` but we convert it to a proper struct.